ui/DesignSystem/Primitive/Emoji.svelte

```diff
@@ -1,10 +1,12 @@
-<script>
+<script lang="ts">
   import twemoji from "twemoji";
 
-  export let emoji = null;
+  export let emoji: string;
 
-  export let size = "regular";
-  export let style = null;
+  type EmojiSize = "small" | "regular" | "medium" | "big" | "huge";
+  export let size: EmojiSize = "regular";
```

Comment:
does `size` here stand for default size?

Reply:
The emoji has a default `size` of `regular` if no `size` prop is provided, but it can be changed to suit the spec like:
```typescript
    <Emoji emoji={'🎷'} size="huge" style="margin-bottom: 1.5rem;" />
```